website/blog/2024/06-10-creating-a-netflix-show-recommender-using-crawlee-and-react/index.md

```diff
@@ -0,0 +1,313 @@
+---
+slug: netflix-show-recommender
+title: 'Building a Netflix show recommender using Crawlee and React'
+tags: [community]
+description: 'Create a Netflix show recommendation system using Crawlee to scrape the data, JavaScript to code, and React to build the front end.'
```

Comment:
Is this the wording we agreed upon? 😄 If I clicked on an article about [recommender systems](https://en.wikipedia.org/wiki/Recommender_system) with scraping (sounds super cool) and got a simple React App, I would be a bit disappointed. 

Netflix is a large player in the realm of recommender systems, with the [Netflix Prize](https://en.wikipedia.org/wiki/Netflix_Prize), their [research papers](https://research.netflix.com/), and stuff... This article is going to have a lot of very strong SEO competition with this name.

Reply:
Yes, we agreed on it. I got the point, and I agree you are right, but this article is not from us. As said in the beginning, it is from one of the community members, and we made it clear that it is not supposed to be that perfect; it is just an app showing something like this can be created through a little work.